packages/engine/src/networking/systems/DataProducerConsumerState.tsx

```diff
@@ -151,24 +151,36 @@ export const DataProducerConsumerState = defineState({
     [
       DataProducerActions.closeProducer,
```

Comment:
Just wondering if this can be renamed to `producerClosed`/`producerClosing` to make it sound similar to other actions in this file.